app/javascript/src/components/Dashboard/Contacts/MenuBar.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const ContactsMenuBar = ({ showMenu }) => {
```

Comment:
Go through the Slack thread very carefully. https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663590973475389?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
changed the component and filename both to SideMenu.jsx and pushed the changes.